### PR TITLE
fix(oracle): power vote calculation

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

Fixes power vote calculation in the oracle end blocker.
https://github.com/umee-network/umee/pull/1845 partially handles that, but in fact introduces a bug.

### CODE REVIEW

File: x/oracle/types/params.go
Comment (toteki) [inline x/oracle/types/params.go:391]:
```suggestion
		return sdkerrors.ErrInvalidRequest.Wrapf("threshold must be bigger than %s and <= 1", minVoteThreshold)
```
In case it changes

File: x/oracle/keeper/params.go
Comment (toteki) [inline x/oracle/keeper/params.go:22]:
```suggestion
// VoteThreshold returns the minimum portion of combined validator power of votes
```

File: x/oracle/keeper/params.go
Comment (toteki) [inline x/oracle/keeper/params.go:31]: Does this TODO need an issue?

File: x/oracle/abci.go
Comment (toteki) [inline x/oracle/abci.go:42]: These two consecutive loops of `for _, v := range k.StakingKeeper.GetBondedValidatorsByPower(ctx)` can be combined

File: x/oracle/abci.go
Comment (toteki) [inline x/oracle/abci.go:68]:
```suggestion
		// Calculate the portion of votes received as an integer, scaled up using the
		// same multiplier as the `threshold` computed above
```

File: x/oracle/abci_test.go
Comment (toteki) [inline x/oracle/abci_test.go:165]:
```suggestion
	// So if only val2 votes, we won't have any prices next block.
```

File: x/oracle/abci.go
Comment (toteki) [reply x/oracle/abci.go:42]: pushed a commit which combines them

**Review by toteki — APPROVED** [review]
Pushed some changes:
- fix simulations
- updates some comments
- combine two for loops with identical iterations

Looks good from my side as soon as test pass - might want to wait for another review from woz before resolving the final conversation

File: x/oracle/types/params_test.go
Comment (rbajollari) [inline x/oracle/types/params_test.go:32]: We shouldn't need this test anymore with the new one at the end of the file covering these cases

**Review by rbajollari — APPROVED** [review]

File: x/oracle/keeper/params.go
Comment (robert-zaremba) [reply x/oracle/keeper/params.go:31]: I have created linear issue for this.

File: x/oracle/types/params_test.go
Comment (robert-zaremba) [reply x/oracle/types/params_test.go:32]: ok, will remove this one.

File: x/oracle/abci.go
Comment (robert-zaremba) [reply x/oracle/abci.go:42]: Yes, another old artifact to improve on.

**Pull request merged**
